recipes/root5/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "root5" %}
+{% set version = "5.34.36" %}
+{% set sha256 = "fc868e5f4905544c3f392cc9e895ef5571a08e48682e7fe173bd44c0ba0c7dcd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: root_v{{version}}.source.tar.gz
+  url: https://root.cern.ch/download/root_v{{version}}.source.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+   number: 0
+   skip: true  # [win]
+   skip: true  # [py3k]
+
+requirements:
+ 
+ build:
+     - toolchain
+     - gcc  # [linux]
+     - python x.x
+     - gsl
+     - fftw
+     - cmake
+     - cfitsio
+     - xorg-libxft  # [linux]
+     - xorg-libxpm  # [linux]
+     - xorg-libsm  # [linux]
+     - jpeg
+     - libpng
+     - zlib
+     - libxml2
+     - mesalib
+    
+ run:
+     - python x.x
+     - gsl
+     - fftw
+     - cfitsio
+     - xorg-libxft  # [linux]
+     - xorg-libxpm  # [linux]
+     - xorg-libsm  # [linux]
+     - jpeg
+     - libpng
+     - zlib
+     - libgcc  # [linux]
+     - libxml2
+     - mesalib
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/root/libCore.so  # [linux]
+    - test -f ${PREFIX}/lib/root/libCore.so  # [osx]
+    - test -f ${PREFIX}/include/root/TROOT.h
+    #- conda inspect linkages -p $PREFIX root5  # [not win]
+    #- conda inspect objects -p $PREFIX root5  # [osx]
+    - root -b -q ${PREFIX}/share/doc/root/tutorials/roofit/rf101_basics.C >& /dev/null
+    - root -b -q ${PREFIX}/share/doc/root/tutorials/fft/FFT.C >& /dev/null
+    - root -b -q ${PREFIX}/share/doc/root/tutorials/physics/PhaseSpace.C >& /dev/null
+    - cd ${PREFIX}/share/doc/root/tutorials/pyroot && python fillrandom.py && python fit1.py >& /dev/null
+    - cd ${PREFIX}/share/doc/root/tutorials/fitsio && root -b -q FITS_tutorial1.C >& /dev/null
+
+about:
+  home: https://root.cern.ch/
+  license: LGPL
+  summary: 'A modular scientific software framework. It provides all the functionalities needed to deal with big data processing, statistical analysis, visualisation and storage. It is mainly written in C++ but integrated with other languages such as Python and R.'
+  license_family: GPL
+  license_file: '{{ environ["SRC_DIR"] }}/LICENSE'
```

Comment:
This looks in `SRC_DIR` by default. So the `environ` part can be dropped.

Reply:
ok